src/features/search/components/SearchHeader/SearchHeader.web.test.tsx

```diff
@@ -22,7 +22,7 @@ jest.mock('react-instantsearch-core', () => ({
   }),
 }))
 
-jest.spyOn(useFeatureFlagAPI, 'useFeatureFlag').mockReturnValue(false)
+const useFeatureFlagSpy = jest.spyOn(useFeatureFlagAPI, 'useFeatureFlag').mockReturnValue(false)
```

Comment:
je splitterai bien en deux ?:
const useFeatureFlagSpy = jest.spyOn(useFeatureFlagAPI, 'useFeatureFlag')
useFeatureFlagSpy.mockReturnValue(false)

Reply:
Why not, j'ai repris un truc existant.